doc/environment.yml

```diff
@@ -0,0 +1,23 @@
+name: ploomber-engine-doc
+
+channels:
+  - conda-forge
+
+
+dependencies:
+  - python=3.10
+  - matplotlib
+  - pandas
+  - pip
+  - pip:
+    - jupyter-book
+    - papermill
+    - ploomber-engine
+    - ipykernel
+    - debuglater
+    - nbclient
+    - ipython
+    - parso
+    - nbformat
+    - pytest
+    - ..
```

Comment:
this last line `..` will install all these packages: https://github.com/ploomber/ploomber-engine/blob/9510173587734e7470eb90d2b1b9d5d97781c6b9/setup.py#L16 there's no need to list them one by one